airbyte-integrations/connectors/destination-clickhouse/src/main/kotlin/io/airbyte/integrations/destination/clickhouse/check/ClickhouseChecker.kt

```diff
@@ -75,6 +73,8 @@ class ClickhouseChecker(
 @Singleton
 class RawClickHouseClientFactory {
     fun make(config: ClickhouseConfiguration): Client {
+        require(!config.hostname.startsWith(PROTOCOL)) { PROTOCOL_ERR_MESSAGE }
```

Comment:
should this come with a check test?

Reply:
I reworked the test. Ironically the original test worked but didn't it didn't work in the real world.